### PR TITLE
Add garden.turtle.Jellybean

### PR DESCRIPTION
**Pull request by turtlegarden**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: garden.turtle.Jellybean.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+app-id: garden.turtle.Jellybean
+runtime: org.gnome.Platform
+runtime-version: master
```
Comment (bbhtt) [inline garden.turtle.Jellybean.yml:4]:
```suggestion
runtime-version: 45
```

Comment (bbhtt) [reply garden.turtle.Jellybean.yml:4]: There is no master branch of runtime on Flathub

Comment (bbhtt) [inline garden.turtle.Jellybean.yml:4]:
```suggestion
runtime-version: "45"
```

There is no master branch of runtime on Flathub

File: garden.turtle.Jellybean.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- "/man"
+- "/share/doc"
+- "/share/gtk-doc"
+- "/share/man"
+- "/share/pkgconfig"
+- "/share/vala"
+- "*.la"
+- "*.a"
+modules:
+- name: jellybean
```
Comment (bbhtt) [inline garden.turtle.Jellybean.yml:29]: Fix these it the desktop file, not essential for the build but nice to have

```
esktop-file-validate garden.turtle.Jellybean.desktop 
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Jellybean"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Inventory"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: error: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" contains an unregistered value "Number"; values extending the format should start with "X-"
garden.turtle.Jellybean.desktop: hint: value "Jellybean;Inventory;Number;" for key "Categories" in group "Desktop Entry" does not contain a registered main category; application might only show up in a "catch-all" section of the application menu
```

File: garden.turtle.Jellybean.yml (hunk 1 above)
Comment (turtlegarden) [reply garden.turtle.Jellybean.yml:4]: Oops, forgot I had that on. Changing now

File: garden.turtle.Jellybean.yml (hunk 2 above)
Comment (turtlegarden) [reply garden.turtle.Jellybean.yml:29]: issue resolved on the repository end

**Comment by bbhtt** [comment]
bot, build garden.turtle.Jellybean

**Pull request closed without merging**